repos/c2sm/packages/py-icon4py/package.py

```diff
@@ -47,7 +55,9 @@ def install(self, spec, prefix):
             args.append('.')
 
         pip = inspect.getmodule(self).pip
-        build_dirs = ['common', 'pyutils', 'testutils', 'atm_dyn_iconam']
+        build_dirs = [
+            'common', 'pyutils', 'testutils', 'liskov', 'atm_dyn_iconam'
```

Comment:
Are this build_dirs present in all the tags listed at the beginning of the package?

Reply:
I think to being with, the `build_dirs` will have all the directories from the first icon4py tag we officially support. From the second release onwards, we will add special conditions to add/remove directories depending on the tag.